src/lib/currentRoutePath.ts

```diff
@@ -0,0 +1,13 @@
+import { parse, join } from "path";
+
+/**
+ * Returns path to current view.
+ * Should be used only inside files in the /pages directory.
+ */
+export const currentRoutePath = () => {
```

Comment:
getCurrentRoutePath

Reply:
Turned out it was causing issues when testing - returned to good ol' plain string